release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -3546,4 +3546,30 @@ $ oc adm release info 4.10.38 --pullspecs
 [id="ocp-4-10-38-upgrading"]
 ==== Updating
 
+To update an existing {product-title} 4.10 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc[Updating a cluster within a minor version by using the CLI] for instructions.
+
+[id="ocp-4-10-39"]
+=== RHSA-2022:7211 - {product-title} 4.10.39 bug fix and security update
+
+Issued: 2022-11-01
+
+{product-title} release 4.10.39, which includes security updates, is now available. The bug fixes that are included in the update are listed in the link:https://access.redhat.com/errata/RHSA-2022:7211[RHSA-2022:7211] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2022:7210[RHBA-2022:7210] advisory.
```

Comment:
Linked pages do not exist (I assume they aren't live yet)

Reply:
Links are live now